extensions/wikia/WikiaMobile/js/topbar.js

```diff
@@ -121,11 +122,11 @@ function (
 	//end search setup
 
 	//profile/login setup
-	if (wkPrfTgl) {
+	if (loginButton && !$loginButton.hasClass('new-login')) {
```

Comment:
I'm going to use classList. 
indexOf could be inaccurate in some cases here, and without support in any other places old Androiders and iOSers couldn't spot the difference :)